packages/@storefront/flux-capacitor/src/core/reducers/data/navigations.ts

```diff
@@ -65,21 +67,30 @@ export const receiveNavigations = (state: State, navigations: Store.Navigation[]
 };
 
 // tslint:disable-next-line max-line-length
-export const selectRefinement = (state: State, { navigationId, index: refinementIndex }: Actions.Payload.Navigation.Refinement) => {
-  if (navigationId && refinementIndex != null) {
-    return {
-      ...state,
-      byId: {
-        ...state.byId,
-        [navigationId]: {
-          ...state.byId[navigationId],
-          selected: state.byId[navigationId].selected.concat(refinementIndex),
-        },
+export const selectMultipleRefinements = (state: State, { navigationId, indices }: Actions.Payload.Navigation.MultipleRefinements) => {
```

Comment:
What happens when `selectMultipleRefinements` is invoked with a payload of just `{ navigationId }` (ie. no `indices`, or an `indices` value which does not have a `length` prop)? Worth defaulting an empty array?

Reply:
We catch this already in the respective action creator.

Reply:
Sounds good, resolving.